### PR TITLE
Add translation for tide/squash-on-merge to contributor-cheatsheet

### PR DESCRIPTION
**Pull request by YuikoTakada**
This PR adds translation for tide/squash-on-merge to contributor-cheatsheet/README-ja.md

**Which issue(s) this PR fixes**:

NONE

### CODE REVIEW

File: contributors/guide/contributor-cheatsheet/README-ja.md [hunk 1]
```diff
@@ -290,6 +290,13 @@ git checkout -b myfeature
 [スカッシュコミット]の主な目的はきれいに読めるgitの履歴や加えられた変更のログを作成することです。通常これはPRの改定の最終段階に行われます。
 コミットをスカッシュするべきかわからない場合は、作業を止めてPRのレビューと承認を担当する他のコントリビューターの判断に任せることをおすすめします。
```
Comment (atoato88) [inline contributors/guide/contributor-cheatsheet/README-ja.md:292]: In [original document](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet#squashing-commits), there is a sentence about `Perform an interactive rebase to choose which commits ...` on second paragraph.

Could you add translation for that?

File: contributors/guide/contributor-cheatsheet/README-ja.md [hunk 2]
```diff
@@ -290,6 +290,13 @@ git checkout -b myfeature
...
 
+```
+git rebase -i HEAD~3
+...
+git push --force
+```
+
+**注意**: レビュー担当者にPRへの `tide/merge-method-squash` ラベルの付与を依頼することも可能です。（このラベルはレビュー担当者が `/label tide/merge-method-squash` コマンドを実行すると付与されます。）`tide/merge-method-squash` ラベルを付与すると、botはこのPRを構成する _すべての_ コミットをまとめ、それにより `LGTM` ラベル（既に適用されている場合）の削除やCIテストの再実行が行われなくなります。
```
Comment (atoato88) [inline contributors/guide/contributor-cheatsheet/README-ja.md:299]: What do you think about raplacing `注意` with `備考` ?
This sentence doen't have mandotary contents for developping, but it's better to know about this.
So, I think `備考` is better.

WDYT? :thinking:

Comment (atoato88) [reply contributors/guide/contributor-cheatsheet/README-ja.md:299]: FYI:
And `Note` is translated to `備考` in kubernetes.io.
For example [here].(https://kubernetes.io/ja/docs/concepts/workloads/pods/#pod%E3%81%A8%E3%81%AF%E4%BD%95%E3%81%8B)

**Comment by atoato88** [comment]
Thank you to PR!
I reviewed this because I updated this file in the past.

File: contributors/guide/contributor-cheatsheet/README-ja.md (hunk 2 above)
Comment (YuikoTakada) [reply contributors/guide/contributor-cheatsheet/README-ja.md:299]: Thanks. I'll fix it.

File: contributors/guide/contributor-cheatsheet/README-ja.md (hunk 1 above)
Comment (YuikoTakada) [reply contributors/guide/contributor-cheatsheet/README-ja.md:292]: Nice catch, thanks.

**Comment by YuikoTakada** [comment]
@atoato88 Thank you for your comments.

File: contributors/guide/contributor-cheatsheet/README-ja.md [hunk 3]
```diff
@@ -290,6 +290,15 @@ git checkout -b myfeature
 [スカッシュコミット]の主な目的はきれいに読めるgitの履歴や加えられた変更のログを作成することです。通常これはPRの改定の最終段階に行われます。
 コミットをスカッシュするべきかわからない場合は、作業を止めてPRのレビューと承認を担当する他のコントリビューターの判断に任せることをおすすめします。
 
+`git rebase` を対話モードで実行して、保持するコミットとまとめるコミットを選択し、ブランチを強制的にプッシュします:
```
Comment (atoato88) [inline contributors/guide/contributor-cheatsheet/README-ja.md:293]: Thank you for adding.
This sentence is not translated with 1 by 1 words, but It's OK for me and maybe most Japanese, I think.

**Comment by atoato88** [comment]
I reviewed updated codesets, and It LGMT :smile: 
/lgtm

**Comment by mrbobbytables** [comment]
Thank you @YuikoTakada for adding it and @atoato88 for doing the review :)
/approve

**Pull request merged**